pkg/driver/driver_azure.go

```diff
@@ -24,15 +24,16 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	v1alpha1 "github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1"
 	"github.com/gardener/machine-controller-manager/pkg/metrics"
 	"github.com/prometheus/client_golang/prometheus"
 	corev1 "k8s.io/api/core/v1"
 
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
-	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-11-01/network"
-	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-11-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
```

Comment:
Should we be using a version later than `2019-07-01` here? Or it doesn't matter?

Reply:
In general it is always a good idea to use the latest one available but for the bug fix, only the Compute API version is relevant.

Reply:
I just used the mentioned azure service sdks.
We could also introduce now the most recent ones? WDYT?

Reply:
sounds good to me.

Reply:
Let's go one with this version of the azure go sdks as they contain the required fixes. Upgrading to the latest ones seems to be more effort because it seems that there are some things have changed. Will do that in a subsequent pr.